src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommand.java

```diff
@@ -73,7 +73,11 @@ public ExtractSubcommand(Supplier<ExtractExecutor> executorSupplier) {
   @Option(
       names = {"--output", "-o"},
       required = true,
-      description = "Output path to which to write the extracted information.")
+      description = {
+        "Output path to which to write the extracted information.",
+        "The output is written into a ZIP file if the output path ends in '.zip'.",
+        "Otherwise, the output path must be an existing directory."
```

Comment:
Would it be better to just accept any path regardless of whether the directory exists or not? We could just create the directory if it doesn't exist.

Reply:
I'd rather not as this could create unwanted directories because the input had a typo.